graviti/openapi/policy.py

```diff
@@ -10,24 +10,24 @@
 from graviti.openapi.requests import open_api_do
 
 
-def get_policy(
+def get_object_policy(
     access_key: str,
     url: str,
     owner: str,
     dataset: str,
     *,
-    action: Optional[str] = None,
+    actions: Optional[str] = None,
     is_internal: Optional[bool] = None,
     expired: Optional[int] = None,
-) -> Dict[str, str]:
-    """Execute the OpenAPI `GET /v2/datasets/{owner}/{dataset}/policy`.
+) -> Dict[str, Any]:
+    """Execute the OpenAPI `GET /v2/datasets/{owner}/{dataset}/policy/object`.
 
     Arguments:
         access_key: User's access key.
         url: The URL of the graviti website.
         owner: The owner of the dataset.
         dataset: Name of the dataset, unique for a user.
-        action: The specific actions including "get" and "put". The default value in the OpenAPI
+        actions: The specific actions including "get" and "put". The default value in the OpenAPI
```

Comment:
should the enum values in the OpenAPI be all uppercase?
like:
- OSS, S3, AZURE
- PUT, GET